packages/node-cli/src/commands/actionresult/CreateProjectActionResult.js

```diff
@@ -0,0 +1,109 @@
+/*
+** Copyright (c) 2019 Oracle and/or its affiliates.  All rights reserved.
+** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+'use strict';
+const assert = require('assert');
+const ActionResult = require('./ActionResult');
+
+class CreateProjectActionResult extends ActionResult {
```

Comment:
We should rename this one to ProjectActionResult to follow the same approach as in the other "ActionsResults".